### DISCUSSION TITLE
Replace vertical sliders/meters with knob controls

### DISCUSSION THREAD

**Discussion by chrisrimple** [jamulussoftware/jamulus#2128] [UI Related]
May be related to #2126.

Most users do not need the fine control (1% increments) provided by vertical sliders/meters in the mixer UI.  Replace those controls with knobs, having values 1-10 (10% increments), and "wrap" the LEDs around the knobs.  This will make it much easier to adjust server-returned volume levels (such as from 60% to 40%, aka 6 to 4) and will also allow far more performers to be visible at one time in the UI.

**Comment by dcorson-ticino-com** [comment]
Although I have nothing against real knobs in a panel, knobs on a screen are a total pain to control.
We can think of other ways of reducing used screen real estate, but please no knobs. 
Personally I would welcome horizontal sliders as for my hands and mouse they are the easiest and fastest to control.

**Reply by chrisrimple** [reply]
If a knob has stops (increments at 1), then drag-n-drop isn't needed - just click on the number to set the knob to that position, which is far simpler than any control that requires dragging.

**Reply by dcorson-ticino-com** [reply]
Of course, but that means many clicks until you get it just right, looking each time to click at the right position instead of a quick drag with a little correction at the end.

**Reply by rdica** [reply]
+1 @dcorson-ticino-com 
Knobs don't solve a problem, at least in my use.
Meters are EXTREMELY useful,

**Reply by gene96817** [reply]
Physical knobs are great. I have yet to experience an easy-to-use mouse control over a knob.
Why not just shrink the length of the meter to save space? When the meter is shortened, also adjust the granularity so that it is easy to repeat a setting. For example, on a very short/compact meter, the slider could be in 20% increments. Further, in this example, it is possible to allow a click above or below the slider setting to select the next setting (click up for +20%, etc.) Then you have the control behavior of a knob for the sliders.

**Reply by gilgongo** [reply]
I think some (most?) DAWs allow click selection and then keyboard control after that. This implies some visual feedback on the fader/knob to know it's selected, then just hit up/down arrows on the keyboard. Not ideal of course, but perhaps a compromise?
